### PR TITLE
Add Turkish Language File

### PR DESCRIPTION
**Pull request by adilarli**

### CODE REVIEW

**Comment by stianst** [comment]
Thanks, we need to find someone from the community that can review this

**Comment by stianst** [comment]
@keycloak-ci-bot test

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: updatePasswordMessage=G\u00FC\u00E7l\u00FC bir \u015Fifre, say\u0131lar, harfler ve sembollerin kar\u0131\u015F\u0131m\u0131ndan olu\u015Fmal\u0131d\u0131r. Tahmin etmesi zor ve ger\u00E7ek bir kelimeye benzemeyen \u015Fifre sadece bu hesap i\u00E7in kullan\u0131l\u0131r.

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: locality = \u015Eehir

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: If 'Tarihi' is used here (which means the **date**) is should also used for the next 2 lines.

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: lastAccess=Son Eri\u015Fim Tarihi

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: expires=Son Kullanma Tarihi

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: grantedPermissions= Verilen \u0130zinler

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: needMyApproval=Onay\u0131m gerekli

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: permissionRequestion=\u0130zin Talepleri

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: not sure about this, is it 	Japanese (Japonca)?
locale_ja=Japonca

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: group=Grup

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (AlicanAkkus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: should be 'Kimlik Sağlayıcılar'

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (AlicanAkkus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: should be 'Kimlik Sağlayıcıları Yönet'

**Review by batuhankok — DISMISSED** [review]

**Comment by stianst** [comment]
@AlicanAkkus @neslihanedes can you review again and approve if you are happy with changes?

**Review by AlicanAkkus — DISMISSED** [review]

**Comment by AlicanAkkus** [comment]
> @AlicanAkkus @neslihanedes can you review again and approve if you are happy with changes?

looks like good. 👍🏻

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: `inResource=Kaynak`

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: `shares=Payla\u015F\u0131m(lar)`

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: `people=\u0130nsanlar`

**Review by neslihanedes — CHANGES_REQUESTED** [review]
I have suggested some capital letter corrections, the rest seems fine :)

**Comment by stianst** [comment]
@AlicanAkkus thanks

@neslihanedes thanks for second review,. @adilarli has pussed changes for capital letters, can you let me know if you are happy with the PR now?

**Review by neslihanedes — APPROVED** [review]

**Review by neslihanedes — APPROVED** [review]

**Comment by neslihanedes** [comment]
@stianst I'm happy with the last changes :)

**Pull request merged**
